### PR TITLE
Implementing real DelimArray<T> for Kotlin SDK

### PR DESCRIPTION
**Pull request by jkaster**
Created a true `DelimArray<T>`

- Updated test for comma-delimited ID parameter passing
- Added DelimArray test

### CODE REVIEW

File: swift/looker/Tests/lookerTests/methodsTests.swift [hunk 1]
```diff
@@ -167,6 +167,15 @@ class methodsTests: XCTestCase {
         XCTAssertEqual("", result, result)
     }
 
+    func testGetAllUsersWithIds() {
+            let ids: [Int64] = [47, 48]
+            let users = try? sdk.ok(sdk.all_users(ids: ids))
+            XCTAssertEqual(users![0].first_name, "Ross")
+            XCTAssertEqual(users![0].last_name, "Bailey")
+            XCTAssertEqual(users![1].first_name, "Rob")
+            XCTAssertEqual(users![1].last_name, "Miller")
```
Comment (rlm1023) [inline swift/looker/Tests/lookerTests/methodsTests.swift:176]: Hah, well this might work for any generic instance 🤔

Comment (jkaster) [reply swift/looker/Tests/lookerTests/methodsTests.swift:176]: Oh sure.

Comment (jkaster) [reply swift/looker/Tests/lookerTests/methodsTests.swift:176]: Dang .. I thought you only had the test for Kotlin. Ok, I'll get to this tomorrow if I don't do it tonight.

Comment (jkaster) [reply swift/looker/Tests/lookerTests/methodsTests.swift]: Ok, it's generic now. Sorry, Rob.

**Pull request merged**